android/app/build.gradle

```diff
@@ -223,6 +223,7 @@ repositories {
 
 dependencies {
     compile project(':react-native-splash-screen')
+    compile project(':react-native-linear-gradient')
```

Comment:
Drop shadows are only supported with `elevation` on android and you can't set the direction of the light so gradients it is.